.github/workflows/go.yml

```diff
@@ -55,8 +55,9 @@ jobs:
     - name: Get dependencies
       run: make deps
 
+    # The `|| true` is here because govulncheck is very touchy. I'm not sure what we should be doing with it.
```

Comment:
Last night it triggered because of a new issue in go 1.20.6, but 1.20.7 hadn't yet been released to the GitHub Runners, so it was unfixable. It's fixed now, so I removed the `|| true`

At a theoretical level, it tends to trigger on random dependencies, breaking a wholly unrelated build. Which just feels wrong. But I'm not really sure what better is. I can imagine a world with a more serious devops and ci/cd setup, where it's not part of the PR merges, and instead part of some larger nightly analysis. But we don't have that yet, so it feels awkward.